### DISCUSSION TITLE
Sorry, but I cannot find a list of rules and RULE_IDs

### DISCUSSION THREAD

**Discussion by leferi99** [ltex-plus/vscode-ltex-plus#76] [Q&A] [answered]
Where is a list of rules and IDs? I want to add for example the "long length of dash may have been used" to the disabled rules, but I don't know what is the RULE_ID to put in settings.json. 

Edit: Now that I look at it again, this example may be a chktex issue

Still, the question is relevant, is there a list somewhere?

**Comment by spitzerd** [comment, marked as answer]
Here is a list: https://community.languagetool.org/rule/list?lang=en
